docs/app/pages/components/components-sections/icon/icon/icon.component.ts

```diff
@@ -1,14 +1,18 @@
-import { Component, Inject } from '@angular/core';
-import { BaseDocumentationSection } from '../../../../../components/base-documentation-section/base-documentation-section';
-import { DocumentationSectionComponent } from '../../../../../decorators/documentation-section-component';
-import { IPlayground } from '../../../../../interfaces/IPlayground';
-import { IPlaygroundProvider } from '../../../../../interfaces/IPlaygroundProvider';
-import { DOCUMENTATION_TOKEN, DocumentationType } from '../../../../../services/playground/tokens/documentation.token';
+import {ChangeDetectionStrategy, Component, Inject} from '@angular/core';
```

Comment:
Could you check your editor settings, perhaps you have something like `prettier` enabled, it seems to be altering spaces in imports, we normally have a space after the bracket: `import { ChangeDetectionStrategy`, otherwise every file you edit from now on will add a lot of unnecessary formatting changes